Tests/AutoTests/AccessebilityTest/accessebility.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect, chromium, Page, Browser, BrowserContext } from '@playwright/test';
+import * as data from "../../configuration/pageObject.json";
+import * as app from "../../Configuration/appConfig.json";
+import headerPage from '../../pageObject/home.page';
+import { injectAxe, getViolations, checkA11y} from 'axe-playwright'
+
+test.describe("Goto maritime-safety-information Home Page ", ()=> {
+  let browser: Browser;
+  let context: BrowserContext;
+  let page: Page;
+  let homepage:headerPage;
+  test.beforeAll(async () => {
+    browser = await chromium.launch();
```

Comment:
You don't need to launch the browser explicitly here as this is just overriding automatic functionality of the framework